### DISCUSSION TITLE
2023/12/10/vscode%E9%85%8D%E7%BD%AE%E8%BF%9C%E7%A8%8B%E6%9C%8D%E5%8A%A1%E5%99%A8/

### DISCUSSION THREAD

**Discussion by giscus** [cyclinbox/cyclinbox.github.io#7] [General]
# 2023/12/10/vscode%E9%85%8D%E7%BD%AE%E8%BF%9C%E7%A8%8B%E6%9C%8D%E5%8A%A1%E5%99%A8/

本文介绍了vscode代码编辑器配置远程服务器插件的方法。

https://wz.anoms.top/2023/12/10/vscode%E9%85%8D%E7%BD%AE%E8%BF%9C%E7%A8%8B%E6%9C%8D%E5%8A%A1%E5%99%A8/

**Comment by Hazelandinstant** [comment]
所以，我们也可以用服务器Matlab的运算能力，在自己电脑的vscode上操作吗？
最近才开始用服务器，我有好多问题呀，多讲讲服务器吧！
比如实验室几个服务器中，有的服务器比较空但是有的比较卡，我如何能“机智地”选择合适的服务器呢？（目前只掌握了htop，并且还只能是每个服务器都打开看看）

**Reply by cyclinbox** [reply]
没有测试过matlab，但是理论上可以用，就相当于是在自己电脑的vscode上写代码，然后在服务器上运行嘛。
关于服务器切换，这个……还真不太清楚😢但我知道如何在向计算节点提交任务的时候选择合适的节点。
不知道你们的服务器使用的作业管理系统是什么，如果作业管理系统是slurm的话，可以用指令 `sinfo -N -l` 查询计算节点的状态
如果是其他的管理系统……等我这几天翻翻资料hhh
